lib/htmldump.js

```diff
@@ -50,7 +50,7 @@ function getArticles (options, res) {
         query: query,
         headers: {
             'user-agent': options.userAgent,
-            host: options.prefix
+            host: options.host
```

Comment:
This should be the domain. When sending requests to the appservers, the Host header indicates the actual domain to fetch info for, not the originating host.